sites/hashdev/src/util/mdxComponents.tsx

```diff
@@ -22,10 +22,6 @@ export const mdxComponents: Record<string, ComponentType<any>> = {
   Typography,
 
   p: (props: TypographyProps<"p">) => {
-    if (!Array.isArray(props.children) && typeof props.children !== "string") {
-      // eslint-disable-next-line react/jsx-no-useless-fragment -- fragment is required by tsc
-      return <>{props.children}</>;
-    }
```

Comment:
I'm not really sure what the intentions of this is either, it's not something [we are doing on the BP site](https://github.com/blockprotocol/blockprotocol/blob/main/site/src/components/mdx/mdx-components.tsx#L197-L199). Happy for us to get rid of it, but might be worth getting thoughts from @nathggns (the git blame suggests it was added as part of https://github.com/hashintel/hash/commit/1e19c3327aaaff1c5fd9b18d150b88ecd8fecbc5)

Reply:
Yeah I have no recollection of why this is there. It was added in https://github.com/hashintel/hash/commit/076e06dee396f6e845aac3ab5d6bfbbd6b922556 which I wrote, but I have no memory of it. If it works, I've no issue changing this.